### PR TITLE
[CGPROD-2204] Parse JSON into Phaser.Geoms

### PR DESCRIPTION
**Pull request by AdamXA**
just the geom parsing stuff in this PR

### CODE REVIEW

File: src/core/loader/particles-loader/geom-parse.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    rectangle: config => new Phaser.Geom.Rectangle(config.x, config.y, config.width, config.height),
+    circle: config => new Phaser.Geom.Circle(config.x, config.y, config.radius),
+    ellipse: config => new Phaser.Geom.Ellipse(config.x, config.y, config.width, config.height),
+    line: config => new Phaser.Geom.Line(config.x1, config.y1, config.x2, config.y2),
+    point: config => new Phaser.Geom.Point(config.x, config.y),
+    polygon: config => new Phaser.Geom.Polygon(config.points),
+    triangle: config => new Phaser.Geom.Triangle(config.x1, config.y1, config.x2, config.y2, config.x3, config.y3),
+};
+
+export const geomParse = geomConfig => geomObjects[geomConfig.type](geomConfig);
```
Comment (FostUK) [inline src/core/loader/particles-loader/geom-parse.js:16]: I like this line.

Comment (AdamXA) [reply src/core/loader/particles-loader/geom-parse.js:16]: what in particular do you like about this line

File: test/core/loader/particles-loader/particle-file.test.js [hunk 2]
```diff
@@ -39,6 +42,54 @@ describe("Particles File", () => {
...
+            geomParse.mockImplementation(() => "mock emit zone");
+            file.xhrLoader = {
+                responseText: testJSON,
+            };
+            file.onProcessComplete = jest.fn();
+            file.onProcess();
+            expect(file.data).toEqual({ emitZone: { source: "mock emit zone" } });
+        });
+
+        test("calls the the Geom parser when a deathZone source is provided", () => {
```
Comment (Leonie-) [inline test/core/loader/particles-loader/particle-file.test.js:69]: Do we need a test to check that the Geom parser is not called when there is niether a death zone or an emit zone?

File: src/core/loader/particles-loader/geom-parse.js (hunk 1 above)
Comment (Leonie-) [reply src/core/loader/particles-loader/geom-parse.js:16]: It is very terse, and works nicely with the object literal above.

File: test/core/loader/particles-loader/particle-file.test.js (hunk 2 above)
Comment (AdamXA) [reply test/core/loader/particles-loader/particle-file.test.js]: added, thanks 👍

**Pull request merged**